homeassistant/components/met/weather.py

```diff
@@ -122,20 +119,24 @@ async def async_setup_entry(
     )
 
 
-def format_condition(condition: str) -> str:
+def format_condition(condition: str | None) -> str | None:
```

Comment:
I'd rather add a check for None on line 190 before using this helper so that we keep the return value narrow here.